File: cache/src/main/java/com/mm/http/cache/IgnoreKey.java
package com.mm.http.cache;

import static java.lang.annotation.RetentionPolicy.RUNTIME;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.Target;

/**
 * Created by : majian
 * Date : 2021/8/24
 */
@Target(ElementType.METHOD)
@Retention(RUNTIME)
public @interface IgnoreKey {
    String[] value() default {"binding"};
}

File: cache/src/main/java/com/mm/http/cache/IgnoreKey.kt
package com.mm.http.cache

/**
 * Created by : majian
 * Date : 2021/8/24
 * 缓存[CacheHelper.key]key值:以url链接和 get query参数 和 post body 参数拼接形成
 *
 */
@Target(AnnotationTarget.FUNCTION)
@Retention(AnnotationRetention.RUNTIME)
annotation class IgnoreKey(vararg val value: String = [""])